docs/deployment/deploy_with_existing_vineyard_cluster.md

```diff
@@ -1,8 +1,22 @@
-# Depoly with Existing Vineyard Cluster
+# Deploy with Existing Vineyard Cluster
 
 If you have already deployed a vineyard cluster, you can easily deploy GraphScope on the existing cluster and reuse the vineyard data such as graph with several GraphScope sessions. This will allow you to load a graph to the existing vineyard cluster and then reuse it with multiple GraphScope sessions, without needing to deploy a separate vineyard cluster for each session.
 
-This doc provides step-by-step instructions on how to do this.
+:::{figure-md}
+
+<img src="../images/session_connect_vineyard_cluster.png"
+     alt="GraphScope sessions connect to an existing vineyard cluster"
+     width="80%">
+
+GraphScope sessions connect to an existing vineyard cluster for data sharing
+:::
+
+The figure above shows that GraphScope sessions can share the data in the same vineyard cluster as the engines are deployed on the same node of the Kubernetes cluster and connected to the same vineyard socket. This is a common use case for vineyard clusters, as it allows multiple sessions to reuse the same graphs as long as the vineyard cluster is alive.
```

Comment:
"as the engines are deployed on the same node of the Kubernetes cluster and connected to the same vineyard socket. This is a common use case for vineyard clusters, as it allows multiple sessions to reuse the same graphs as long as the vineyard cluster is alive." => "as the engines in different sessions are deployed on the same node within the Kubernetes cluster and connected to the same vineyard socket. Multiple sessions can reuse the same graph as long as the vineyard cluster is alive. This is a common use pattern of vineyard on Kubernetes."

Reply:
@dashanji Could let the chatgpt help to refine the grammar and spelling for you